tools/pythonpkg/tests/fast/numpy/test_numpy_new_path.py

```diff
@@ -0,0 +1,32 @@
+""" The support for scaning over numpy arrays reuses many codes for pandas.
+    Therefore, we only test the new codes and exec paths.
+"""
+
+import numpy as np
+import duckdb
+
+class TestScanNumpy(object):
+    def test_scan_numpy(self, duckdb_cursor):
+        z = np.array([1,2,3])
```

Comment:
Could we add some more tests with:

* Different data types (e.g. objects, timedeltas, etc)
* Empty arrays
* Null values in arrays
* Dictionaries with mixed types (e.g. `{'a': np.array([1, 2, 3]), 'b': 42}`
* Arrays with mixed types (e.g. `[np.array([1, 2, 3]), 42]`
* 3+-dimensional numpy arrays (does that exist?)
* Array of 2-dimensional numpy arrays
* Dictionary with 2-dimensional numpy arrays